lib/logstash/outputs/datadog_metrics.rb

```diff
@@ -9,8 +10,7 @@
 
 # Default `queue_size` and `timeframe` are low in order to provide near realtime alerting.
 # If you do not use Datadog for alerting, consider raising these thresholds.
-
-class LogStash::Outputs::DatadogMetrics < LogStash::Outputs::Base
+module LogStash module Outputs class DatadogMetrics < LogStash::Outputs::Base
```

Comment:
not sure I'd like to do this change in this pr. If decided that this or a similar approach is desirable, it should be a mass update on all plugins. thoughts?